local/blockchain.go

```diff
@@ -137,7 +139,7 @@ func (ln *localNetwork) installCustomVMs(
 			if _, err := platformvm.Codec.Unmarshal(subnetTxBytes, &subnetTx); err != nil {
 				return nil, fmt.Errorf("couldn not unmarshal tx for subnet %q: %w", subnetID.String(), err)
 			}
-			pTXs[subnetID] = &subnetTx
+			pTXs = append(pTXs, subnetID)
```

Comment:
it seems most part of this for is not needed anymore (obtaining the tx, unmarshall, etc)

Reply:
probably update the comments and also the var name with txIDs ? subnetTxIDs ?

Reply:
You're very right with the first comment, thanks.
Regarding the second, I'd like to leave it as subnetID, it's a good reminder that the subnet ID is actually also the tx ID. Finally, pTXs was also used inside avalanchego, so I'd like to keep it this way.